sky/backends/backend_utils.py

```diff
@@ -1787,28 +1787,41 @@ def _is_local_cluster(record):
         f'[bold cyan]Refreshing status for {len(records)} cluster{plural}[/]',
         total=len(records))
 
-    terminated_clusters = []
-
     def _refresh_cluster(cluster_name):
         record = _update_cluster_status(cluster_name,
                                         acquire_per_cluster_status_lock=True)
-        if record is None:
-            terminated_clusters.append(cluster_name)
         progress.update(task, advance=1)
         return record
 
     cluster_names = [record['name'] for record in records]
     with progress:
         updated_records = subprocess_utils.run_in_parallel(
             _refresh_cluster, cluster_names)
-    if terminated_clusters:
-        plural = 's were' if len(terminated_clusters) > 1 else ' was'
-        cluster_str = ', '.join(repr(name) for name in terminated_clusters)
-        yellow = colorama.Fore.YELLOW
-        reset = colorama.Style.RESET_ALL
-        logger.warning(f'{yellow}The following cluster{plural} terminated on '
-                       'the cloud and removed from the cluster table: '
-                       f'{cluster_str}{reset}')
+
+    # Show information for removed clusters.
+    autodown_clusters, remaining_clusters = [], []
+    for i, record in enumerate(records):
+        if updated_records[i] is None:
+            if record['to_down']:
+                autodown_clusters.append(cluster_names[i])
+            else:
+                remaining_clusters.append(cluster_names[i])
+
+    yellow = colorama.Fore.YELLOW
+    light = colorama.Fore.LIGHTBLACK_EX
```

Comment:
Consider no color or a different color. A bit hard to read on a dark theme.

Reply:
Changed to `Style.BRIGHT` instead. Thanks!